instant-xml/src/de.rs

```diff
@@ -353,6 +372,9 @@ impl<'xml> Iterator for Context<'xml> {
                     false => return Some(Err(Error::UnexpectedToken(format!("{token:?}")))),
                     true => {}
                 },
+                Ok(Token::Comment { text, .. }) => {
+                    self.records.push_back(Node::Comment(text.as_str().into()));
```

Comment:
Actually, not sure we have a reason to preserve these? Couldn't we just skip them for now, which would avoid the need for the `Deserializer` changes?

Reply:
Ah yeah, good point...

Unless we also allow serializing /// comments in structs into the xml. Then I would say we should keep this to maintain the 1:1 mapping.

Do you think it makes sense to add ser support for comments?

Reply:
Unless you have a use case for it in the near future, I think we shouldn't worry about it for a while. KISS!